### PR TITLE
Action

### PR DESCRIPTION
**Pull request by IdanTo1**
**pay attention!**
this pull relies on approval of [printdb](https://github.com/IdanTo1/spl-assignment4/pull/3)!
first we'll approve and marge printdb, and then review this pull

### CODE REVIEW

**Review by Hroy02 — APPROVED** [review]
Looks good

File: action.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    db_conn = sqlite3.connect(DB_FILE_NAME)
+    db_cursor = db_conn.cursor()
+    with open(actions_file, 'r') as f:
+        for line in f.readlines():
+            args = line.split(',')
+            args = [arg.strip() for arg in args]
+            activity = Activity(*args)
+            curr_quantity = _Products(db_cursor).get_quantity(activity.product_id)
+            new_quantity = curr_quantity + int(activity.quantity)
+            if new_quantity > 0:
```
Comment (Hroy02) [inline action.py]: Change the > to >= to accomodate for exact removals (do you also need to remove it if the new quantity is exactly 0?)

**Review by Hroy02 — CHANGES_REQUESTED** [review]
One Letter off perfection!

File: action.py (hunk 1 above)
Comment (IdanTo1) [reply action.py]: Absolutely right, I'll fix it

**Review by Hroy02 — APPROVED** [review]
Looks great!

**Pull request merged**